internal/rename/namespace/executor.go

```diff
@@ -0,0 +1,34 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2019 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+package namespace
+
+import (
+	"github.com/eclipse-iofog/iofogctl/internal/config"
+	"github.com/eclipse-iofog/iofogctl/pkg/util"
+)
+
+func Execute(name, newname string) error {
+	if name == "" || name == "default" {
+		return util.NewError("Cannot rename default or nonexistant namespaces")
+	}
+
+	_, err := config.GetNamespace(name)
+	if err != nil {
+		return err
+	}
+
+	config.RenameNamespace(name, newname)
```

Comment:
need a flush?

Reply:
Done within the config, can move that around tho